axlearn/common/input_lm.py

```diff
@@ -21,6 +21,9 @@
 # Value of "target_labels" which will be ignored in seq2seq processing.
 SEQ2SEQ_IGNORE_TARGET_LABEL = -1
 
+seed = os.environ.get("DATA_SEED")
+seed = int(seed) if seed is not None else None
+tf.random.set_seed(seed)
```

Comment:
would this line work if seed is `None`

Reply:
Yes it will work as it's currently working, without any set seed.